Minio/Credentials/AssumeRoleBaseProvider.cs

```diff
@@ -117,15 +110,15 @@ internal virtual async Task<HttpRequestMessageBuilder> BuildRequest()
 
     public override async Task<AccessCredentials> GetCredentialsAsync()
     {
-        if (Credentials != null && !Credentials.AreExpired()) return Credentials;
+        if (Credentials?.AreExpired() == false) return Credentials;
```

Comment:
The same comment as before for the unnecessary ` == false`.

Reply:
Yes because of nullable